platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformWithDependenciesIntegrationTest.groovy

```diff
@@ -750,10 +752,8 @@ project(':common') {
         )
         outputContains("result = [app.txt, lib.jar.txt, common.jar.txt]")
 
+
         when:
-        if (!GradleContextualExecuter.isConfigCache()) {
```

Comment:
How come the CC test does not fail now? I would expect this deprecation to turn into a failure

Reply:
I fixed the test to remove the ambiguity, so there is no problem.  This wasn't a test of the ambiguity, it was a test of `ArtifactView` reselecting properly that just happened to involve ambiguity.